### PR TITLE
Add contributing guidelines and WIP style guide

### PR DESCRIPTION
**Pull request by noramullen1**
I'm not 100% sure on how we keep this on GitHub and not in the live site/Gitbook--probably something with Gitbook configuration we need to figure out.

### CODE REVIEW

**Review by matthewhelmke — APPROVED** [review]
This LGTM

File: contributing/contributing.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### Make changes
+Modify the documentation locally on your machine. When you commit, write a meaningful and clear commit message, such as:
+
+`git commit -m 'Fixes a typo on the Update Documentation page'`
+
+Then, push your changes to your fork on GitHub.
+
+### Submit a pull request
+Use the GitHub web interface to create a pull request (PR) for review.
```
Comment (kelseiv) [inline contributing/contributing.md]:
```suggestion
Use the GitHub web interface to [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) (PR) for review.
```

File: contributing/style-guide.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Pinot documentation style guide
+
+The guidelines below are specific to Apache Pinot's documentation. 
+ 
+ For general style questions or guidance on tpics not covered here, see the [Google style guide](https://developers.google.com/style)
```
Comment (kelseiv) [inline contributing/style-guide.md]:
```suggestion
 For general style questions or guidance on topics not covered here, see the [Google developer documentation style guide](https://developers.google.com/style) 
```

File: contributing/style-guide.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Pinot documentation style guide
+
+The guidelines below are specific to Apache Pinot's documentation. 
+ 
+ For general style questions or guidance on tpics not covered here, see the [Google style guide](https://developers.google.com/style) 
+
+## Star-tree index
+When describing the index type, use star-tree (adjective) or star tree (noun). StarTree refers to the company only.
```
Comment (kelseiv) [inline contributing/style-guide.md:8]: curious is star tree noun a thing? if we haven't run into a specific use case, maybe best not to encourage/address this usage

**Review by kelseiv — APPROVED** [review]

File: contributing/style-guide.md (hunk 3 above)
Comment (noramullen1) [reply contributing/style-guide.md:8]: I have but only in one spot: https://docs.pinot.apache.org/basics/indexing/star-tree-index#definitions (first sentence here)

What do you think?

Comment (kelseiv) [reply contributing/style-guide.md:8]: Maybe 
The star-tree index stores data in a structure that consists of the following properties:


(and then under the graphic)
Star-tree index structure

Comment (noramullen1) [reply contributing/style-guide.md:8]: updated!

**Comment by noramullen1** [comment]
> I'm not 100% sure on how we keep this on GitHub and not in the live site/Gitbook--probably something with Gitbook configuration we need to figure out.

I've since figured out it's in the a Gitbook yaml file which we've just created. For now, we're letting the docs contributing guide live on the Gitbook site as well as GitHub.

**Pull request merged**